inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/StreamSourceFieldEntityMapper.java

```diff
@@ -59,6 +59,13 @@ public interface StreamSourceFieldEntityMapper {
      */
     int deleteAll(@Param("sourceId") Integer sourceId);
 
+    /**
+     * Physically delete all stream source fields based on inlong group ids
```

Comment:
The java doc was incorrect.

`Physically delete all stream source fields based on inlong group id and inlong stream id`

Reply:
Done.